src/win7/win7-uvc.cpp

```diff
@@ -370,6 +370,8 @@ namespace librealsense
         {
             while (_keep_pulling_interrupts)
             {
+                if (_device->deviceData.ctrl_if.bEndpointAddress < 0x80)
```

Comment:
Need to not run poll_interrupts in this case

Reply:
I refactored and check it in init phase